### ISSUE TITLE
Incorrect code rendered

### ISSUE THREAD

**Issue by lpil** [ruby-formatter/rufo#49]
Input:
```ruby
context 'no sidecar/archive' do
  it 'uploads destination master to the specified destination' do
    expect_any_instance_of(Uploader::Null) # rubocop:disable RSpec/AnyInstance
      .to receive(:upload)
      .with([file_path, 'HON_TEST001_010.mxf'])
      .and_return(Uploader::Result.new(success: true))
  end
end
```

Output:

```ruby
context 'no sidecar/archive' do
  it 'uploads destination master to the specified destination' do
    expect_any_instance_of(Uploader::Null)# rubocop:disable RSpec/AnyInstance
.to receive(:upload)
)
)
  end
end
```

**Comment by mjago** [comment]
Fixed with #50 - Thanks for reporting!

**Comment by lpil** [comment]
Thanks :)
